migration/migrator/data/course_tables.sql

```diff
@@ -1528,12 +1528,14 @@ CREATE TABLE public.polls (
     poll_id integer NOT NULL,
     name text NOT NULL,
     question text NOT NULL,
-    status text NOT NULL,
+    status text,
     release_date date NOT NULL,
     image_path text,
     question_type character varying(35) DEFAULT 'single-response-multiple-correct'::character varying,
     release_histogram character varying(10) DEFAULT 'never'::character varying,
-    release_answer character varying(10) DEFAULT 'never'::character varying
+    release_answer character varying(10) DEFAULT 'never'::character varying,
+    duration integer DEFAULT 0,
+    end_date timestamp with time zone DEFAULT '1900-02-01 00:00:00-05'::timestamp with time zone NOT NULL
```

Comment:
Also, what's the purpose of "duration" if a poll already has "end_date (time)", we can just calculate the difference between "now" and when the poll is ended... this way the poll's duration wouldn't reset if an instructor wants to edit a poll's other attributes like title or questions.

Reply:
we get the end_Date based on duration by appending the duration given by input to DateTimeNow(). The reason we can't just do this with end_date is we need to hold the values the instructor gives for the duration and set the endDate based on when they click "accepting poll responses"

Reply:
> Would be nice if the variable name for this can also capture the connotation of "time" instead of "date".

I will change the end_date to endTime to make it more intuitive.